File: BE/DreamyShop.Domain.Shared/Dtos/ProductVariantDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DreamyShop.Domain.Shared.Dtos
{
    public class ProductVariantDto
    {
        public int Id { get; set; }
        public int ProductId { get; set; }
        public string SKU { get; set; }
        public bool IsVisibility { get; set; }
        public bool IsActive { get; set; }
        public string Description { get; set; }
        public string ThumbnailPicture { get; set; }
        public int Quantity { get; set; }
        public double Price { get; set; }
    }
}

File: FE/src/app/shared/models/product-variant.dto.ts
export interface ProductVariantDto {
  attribute_names: Array<string>;
  sku: string;
  quantity: number;
  price: number;
  thumbnail_picture: string;
}

export interface ProductVariantRequestDto {
  attributeNames: Array<string>;
  sKU: string;
  quantity: number;
  price: number;
  thumbnailPicture: string;
}
